test/e2e/input-field.test.js

```diff
@@ -2,13 +2,74 @@ const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlInputFieldPage = require('./pages/vl-input-field.page');
 
 describe('vl-input-field', async () => {
-    const vlInputFieldPage = new VlInputFieldPage(driver);
+	const vlInputFieldPage = new VlInputFieldPage(driver);
 
-    before(async () => {
-        return vlInputFieldPage.load();
-    });
+	before(async () => {
+		return vlInputFieldPage.load();
+	});
+
+	it ('Als gebruiker kan ik text inputten in een inputfield', async() => {
+		const inputText = 'https://webcomponenten.omgeving.vlaanderen.be/doc/index.html';
+		const inputField = await vlInputFieldPage.getInputField();
+		await inputField.setInputValue(inputText);
+		await assert.eventually.equal(inputField.getInputValue(), inputText);
+	});
+
+	it('Als gebruiker zie ik het onderscheid tussen een gewoon inputfield en een block inputfield', async() => {
+		const inputFieldNotBlock = await vlInputFieldPage.getInputField();
+		await assert.eventually.isFalse(inputFieldNotBlock.isBlock());
+		const inputFieldBlock = await vlInputFieldPage.getInputFieldBlock();
+		await assert.eventually.isTrue(inputFieldBlock.isEnabled());
+	});
+
+	it('Als gebruiker zie ik het onderscheid tussen een gewoon inputfield en een error inputfield', async() => {
+		const inputFieldNotError = await vlInputFieldPage.getInputField();
+		await assert.eventually.isFalse(inputFieldNotError.isBlock());
```

Comment:
Moet hier dan niet `isError` gebruikt worden?
```suggestion
		await assert.eventually.isFalse(inputFieldNotError.isError());
```